crates/yttrium/src/chain_abstraction/api/route.rs

```diff
@@ -42,13 +43,19 @@ pub struct RouteResponseAvailable {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
+pub struct Empty;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
 pub struct RouteResponseNotRequired {
     #[serde(rename = "transactions")]
-    _flag: [Infallible; 0],
+    _flag: Empty,
```

Comment:
Does this work? I would expect you need `[Empty; 0]`?

Reply:
It works but will try the above